dapp-test/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <title>Dapp Test</title>
+    <style>
+      * {
+        padding: 0;
+        margin: 0;
+      }
+      canvas {
+        background: #050d4f;
+        display: block;
+        margin: 0 auto;
+      }
+    </style>
+  </head>
+  <body>
+    <script type='text/javascript' src="./massa-wallet-provider.js"></script>
+    <script type='text/javascript' src="./massa-wallet-provider-content-script.js"></script>
+    <script type='module'>
+
+      // this is the wallet extension class
+      class SpaceXWalletExtension extends window.injected.ContentScriptProvider {
```

Comment:
if the purpose of dapp-test is to test the whole library, please consider adding a npm run test command or npm run e2e that automate testing and include this in the CI to get the most of this work.

Reply:
outdated comment